usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -224,3 +224,38 @@ def get_transaction_ids_agg(keyword, size=10000):
         for result in response['aggregations']['results']['buckets']:
             results.append(result['key'])
         yield results
+
+
+def get_sum_and_count_aggregation_results(keyword):
+    index_name = '{}-'.format(TRANSACTIONS_INDEX_ROOT)+'*'
+    query = {"query": {"query_string": {"query": keyword}},
+             "aggs": {
+              "prime_awards_obligation_amount": {
+                 "sum": {
+                    "field": "transaction_amount"
+                 }
+              },
+              "prime_awards_count": {
+                 "value_count": {
+                    "field": "transaction_id"
+                 }
+              }
+              }, "size": 0}
+    found_result = 0
+    while not found_result == 10:
+        try:
+            response = CLIENT.search(index=index_name, body=query)
+            found_result += 1
```

Comment:
Need to move this outside of the try:except block. it is possible the first statement errors and found_result never gets incremented